dbms/src/Formats/CSVRowInputStream.cpp

```diff
@@ -488,4 +488,58 @@ void registerInputFormatCSV(FormatFactory & factory)
     }
 }
 
+void registerChunkGetterCSV(FormatFactory & factory)
+{
+    factory.registerChunkGetter("CSV", [](
```

Comment:
Could you make these just plain function instead of anonymous lambdas? That's easier to navigate and also has less indentation.